Documentation/ceph-filesystem-crd.md

```diff
@@ -115,7 +115,10 @@ The pools allow all of the settings defined in the Pool CRD spec. For more detai
 
 * `metadataPool`: The settings used to create the filesystem metadata pool. Must use replication.
 * `dataPools`: The settings to create the filesystem data pools. If multiple pools are specified, Rook will add the pools to the filesystem. Assigning users or files to a pool is left as an exercise for the reader with the [CephFS documentation](http://docs.ceph.com/docs/master/cephfs/file-layouts/). The data pools can use replication or erasure coding. If erasure coding pools are specified, the cluster must be running with bluestore enabled on the OSDs.
-* `preservePoolsOnDelete`: If it is set to 'true' the pools used to support the filesystem will remain when the filesystem will be deleted. This is a security measure to avoid accidental loss of data. It is set to 'false' by default. If not specified is also deemed as 'false'.
+* `preserveFsOnDelete`: If it is set to 'true' the filesystem will remain on disk when the Filesystem CRD is deleted. This is a security measure to avoid accidental loss of data. It is set to 'false' by default. If not specified is also deemed as 'false'.
```

Comment:
```suggestion
* `preserveFsOnDelete`: If it is set to 'true' the filesystem will remain when the Filesystem CRD is deleted. This is a security measure to avoid accidental loss of data. It is set to 'false' by default. If not specified is also deemed as 'false'.
```

The FS is not on the disk per se, Ceph stores data directly on a block device, and CephFS implementation "just" abstracts those block and present them with a FS interface.

Reply:
Also, this feels critical enough to be "true" by default, right?

Reply:
We can't change the default of `preservePoolOnDelete`, and I'd suggest we keep the same default as the pool.

Reply:
I'm still leaning toward not delete the fs by default. If we realize things should have been different then I'd rather change and make the parameters less consistent with each other. Simply keeping something we know is incorrect just for the sake of consistency is not a good idea IMO...

Reply:
@leseb Thoughts on [this comment](https://github.com/rook/rook/pull/6495#pullrequestreview-518254071)?

Reply:
@travisn I'm fine with this compromise 👍

Reply:
Ack sounds good. Let's go with this.